File: UETshare/src/main/java/com/example/uetshare/response/dto/NotificationCommentDto.java
package com.example.uetshare.response.dto;

import com.example.uetshare.entity.ActionType;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Calendar;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class NotificationCommentDto {

    private Long id;
    private Long comment_id;
    private String username;
    private String avatar;
    private Boolean seen;
    private ActionType action_type;
    private Calendar time;

}

File: app/src/main/java/ie/app/uetstudents/Entity/notifications_comment/put/response/NotificationCommentDto.kt
package ie.app.uetstudents.Entity.notifications_comment.put.response

data class NotificationCommentDto(
    val action_type: String,
    val avatar: String,
    val comment_id: Int,
    val id: Int,
    val seen: Boolean,
    val time: String,
    val username: String
)